wcivf/apps/elections/templates/elections/includes/_election_list.html

```diff
@@ -1,14 +1,16 @@
 {% load i18n %}
 {% regroup elections by election_date as elections_by_type %}
 {% for election_group in elections_by_type %}
-    <h3>
+    <h3 class="grouper-date">
         {{ election_group.grouper }}
     </h3>
-    {% for election in election_group.list %}
-        <ul style="list-style: none;">
-            <li>
-                <a href="{{ election.get_absolute_url }}">{{ election.nice_election_name }}</a>
-            </li>
-        </ul>
-    {% endfor %}
+    <div id="election-urls" >
```

Comment:
Should this be using `class` as I thought that an HTML `id` was meant to be unique?

Reply:
You're right, thanks for catching.